core/src/main/scala/cats/data/Validated.scala

```diff
@@ -265,6 +265,30 @@ sealed abstract class Validated[+E, +A] extends Product with Serializable {
       case Valid(a)   => Right(a)
     }
 
+  /**
+   * Converts the value to an EitherT[F, E, A]
+   *
+   * Example:
+   * {{{
+   * scala> import cats.syntax.all._
+   * scala> given ExecutionContext = ExecutionContext.global
```

Comment:
I believe this is Scala 3 syntax, please replace it with the Scala 2 equivalent (which I assume doc comments are written in)